chapter10/test_debugging/e2e/contacts.e2e-spec.ts

```diff
@@ -61,7 +61,7 @@ describe('contacts test', () => {
 
   fit('should open the dialog with waitForAngular', async () => {
```

Comment:
@cnishina how about `fit`?

Reply:
whoops, that's my fault.

Reply:
It should be fixed. The `fit` should be `it`. I'll get to it tonight.